lib/src/working_copy.rs

```diff
@@ -310,8 +311,8 @@ pub enum SnapshotError {
     #[error("New file {path} containing {size} exceeds configured limit {max_size}")]
     NewFileTooLarge {
         path: PathBuf,
-        size: u64,
-        max_size: u64,
+        size: HumanByteSize,
+        max_size: HumanByteSize,
```

Comment:
nit: for the actual `size` which wouldn't be divisible by 1024, I think the exact byte size is better.

Reply:
The `Display` impl uses an approximate float presentation, which I think is more intelligible for large sizes. Do we really need to be precise here?

Reply:
I just tried with a file slightly exceeds the limit, and `1.000001MiB` looked a bit weird to me. And it looks like a precise value with that number of digits, but it isn't because we're using floating point.

Maybe we can show both exact and approximate value?
```
New file data.blob containing 1073741825B (~1.0MiB) exceeds configured limit 1MiB
```

Reply:
Tweaked wording and added "~". I just don't see the value of counting out the exact number of bytes here, though. What is the user going to do with that information?

Reply:
> I just don't see the value of counting out the exact number of bytes here, though.

Me neither. I just think "1.0MiB exceeds 1.0MiB" would sound odd.

Reply:
I agree; "~1.0MiB exceeds 1.0MiB" seems a lot better, though.